test/acceptance/features/verifyExamples.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Verify examples provided in Service Binding Operator github repository
+
+    As a user of Service Binding Operator
+    I want to verify if all the examples given are vouching for the mentioned features
+
+    Background:
+        Given Namespace [TEST_NAMESPACE] is used
+        * Service Binding Operator is running
+
+    @example
+    # https://github.com/redhat-developer/service-binding-operator/blob/master/examples/route_k8s_resource/README.md
+    Scenario: Bind an application to a openshift route
```

Comment:
@pratikjagrut Thanks for adding this, however I have some notes.

While this scenario shares the important piece, that is binding the route to an application, It actually does not verify the the steps of the `route_k8s_resource` example. The point of the story is to verify the exact steps (or as close as possible) in the example.

For instance, the example uses the Node.js application to bind a route with, however this scenario uses the generic test application. The scenario needs to follow the steps stated in the example (as close as possible) to really verifying it.

Reply:
Okay I get it.